src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaService.java

```diff
@@ -0,0 +1,45 @@
+package com.solr.clientwrapper.domain.service;
+
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solr.clientwrapper.domain.dto.solr.SolrFieldDTO;
+import com.solr.clientwrapper.domain.dto.solr.SolrSchemaDTO;
+import com.solr.clientwrapper.domain.port.api.SolrSchemaServicePort;
+
+@Service
+@Transactional
+public class SolrSchemaService implements SolrSchemaServicePort {
+
+	private final Logger log = LoggerFactory.getLogger(SolrSchemaService.class);
+	
+	HttpSolrClient solrClient = new HttpSolrClient.Builder("http://localhost:8983/solr").build();
```

Comment:
The url should be taken from the properties file. Please check Collection_CRUD branch for more details

Reply:
Yeah, that code has been replaced now. So, done.